server.lua

```diff
@@ -306,6 +306,7 @@ RegisterNetEvent('qb-vehicleshop:server:sellShowroomVehicle', function(data, pla
         local vehicle = data
         local vehiclePrice = QBCore.Shared.Vehicles[vehicle]['price']
         local commission = round(vehiclePrice * Config.Commission)
+        local bossmenu = round(vehiclePrice * Config.Commissionboss)
```

Comment:
This needs to be local bossmenu = round(vehiclePrice * Config.bossmenu) - per your changes to the config.lua in this pull